### PR TITLE
Add roles yaml section to Vulnerability Management WG

### PR DESCRIPTION
**Pull request by beyhan**
This is need for the inactive user automation to not suggest users for deletion who has roles. cc @paulcwarren

### CODE REVIEW

File: toc/working-groups/vulnerability-management.md [hunk 1]
```diff
@@ -31,10 +31,29 @@ Provides discreet management of security vulnerabilities issues relevant for act
...
+
+
+```yaml
+name: Vulnerability Management
+execution_leads:
+- name: Monty Ijzerman
+  github: ijzerman
+- name: Khushboo Soni
+  github: skhushboo-vm
+- name: Mathias Essenpreis
```
Comment (paulcwarren) [inline toc/working-groups/vulnerability-management.md:50]: No longer active

File: toc/working-groups/vulnerability-management.md [hunk 2]
```diff
@@ -31,10 +31,29 @@ Provides discreet management of security vulnerabilities issues relevant for act
...
+  github: Essenpreis
+- name: Prateek Mehrotra
```
Comment (paulcwarren) [inline toc/working-groups/vulnerability-management.md:52]: No longer active

**Review by paulcwarren — APPROVED** [review]

**Review by paulcwarren — CHANGES_REQUESTED** [review]
Can we remove the inactive users please?

File: toc/working-groups/vulnerability-management.md (hunk 1 above)
Comment (beyhan) [reply toc/working-groups/vulnerability-management.md]: Done

File: toc/working-groups/vulnerability-management.md (hunk 2 above)
Comment (beyhan) [reply toc/working-groups/vulnerability-management.md]: Done

**Comment by beyhan** [comment]
I'm going to merge this because org automation wasn't in place and no roles were given to the removed people.

**Review by ameowlia — APPROVED** [review]

**Pull request merged**